sklearn/inspection/_partial_dependence.py

```diff
@@ -269,17 +288,31 @@ def partial_dependence(estimator, X, features, response_method='auto',
         Please see :ref:`this note <pdp_method_differences>` for
         differences between the 'brute' and 'recursion' method.
 
+    individual : True, False or 'both', default=False
+        Whether to return averaged or individual partial dependence
+
+        .. versionadded:: 0.23
+
     Returns
     -------
-    averaged_predictions : ndarray, \
+    predictions : ndarray or tuple of ndarray
```

Comment:
@NicolasHug do you think it could be good to get a Bunch instead of a tuple of `ndarray`.

Reply:
Returning a Bunch would make give us the flexibility to make evolutions to this tool to optionally compute more stuff i the future such as some kinds of upper and lower confidence bounds for the average (e.g. via bootstrapping for instance).

The problem is that we would break the backward compat...

Reply:
A proposal is to have a FutureWarning to return a Bunch in the PD case in 2 versions.

Reply:
A Bunch is returned for kind=individual/both in https://github.com/scikit-learn/scikit-learn/pull/16619/commits/ff677a065798e097439387819ffb7f5c193ce32e. Future warning added in https://github.com/scikit-learn/scikit-learn/pull/16619/commits/bce815cdc2f7443d6b644820cc08a45fdbb9c13c